README.md

```diff
@@ -86,12 +87,28 @@ source $HOME/pythonvirtenvs/capacity-status-env/bin/activate
 Now run the following to start the application in the root directory of the
 project:
 
+make project-start
 make api-start
 
-The application is configured to run on https://localhost:8443/apidoc/
+The application is configured to run on http://localhost:8000/apidoc/
 
 Navigate here in a browser to see all available endpoints and routes.
 
+## Unit testing
+
+### Creating unit tests
+
+* Ensure test files are created in a relevant sub-directory under the test directory of the app under test.
+* Ensure all test files are prefixed with "test_" and match the name of the module/class that this test file tests.
+* Ensure that import unittest is brought into the test file.
+* Base your test class on unittest.TestCase.
+* Add the new test file to the __init__.py file so that the unit test runner picks the new tests up.
+
+### Running the unit tests
+
+The unit tests can be run by executing the following command:
+    ./manage.py test app
```

Comment:
I think the run here needs to be ./application/manage.py test api

Reply:
Actioned.